app/bingey-ui/src/components/WatchlistsControler/WatchlistAdd/WatchlistAdd.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import {
+  Button,
+  ListItem,
+  ListItemIcon,
+  ListItemText,
+  TextField,
+} from '@material-ui/core';
+import Slide from '@material-ui/core/Slide';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogUI from '@material-ui/core/Dialog';
+import './WatchlistAdd.scss';
+import AddCircleIcon from '@material-ui/icons/AddCircle';
```

Comment:
It would be best to import everything from `@material-ui/core` with named exports. Also, separating local imports could reduce possible confusion